postgres/assets/configuration/spec.yaml

```diff
@@ -427,6 +427,22 @@ files:
           value:
             type: number
             example: 3500
+    - name: database_metadata
+      description: Configure collection of database metadata
+      options:
+        - name: collect_settings
+          description: |
+            Enable collection of pg_settings. Requires `dbm: true`.
+          value:
+            type: boolean
+            example: true
+        - name: settings_collection_interval
+          description: |
+            Set the database settings collection interval (in minutes). Each collection involves a single query to
```

Comment:
> in minutes

None of the other collection intervals are in minutes, this seems confusing to customers.

Reply:
yeah I did minutes bc ... writing 300 for the default felt weird, but I can change it to seconds